tools/releaser/scripts/breaking-changes.sh

```
@@ -14,7 +15,10 @@ go install github.com/joelanford/go-apidiff@latest > /dev/null
 GIT_BASE_REF=${GIT_BASE_REF:-git rev-parse head || echo}
 
 echo "Running breaking changes check for $GIT_BASE_REF"
-BREAKING_CHANGES=$("$GOPATH/bin/go-apidiff" "$GIT_BASE_REF" --compare-imports="false" --print-compatible="false" --repo-path="../")
+
+pushd "$script_path/../../../" || exit ## workaround for --repo-path="../" not working
```

Temp change as workaround for --repo-path silently failing